website/frontend/src/components/get-app/GetApp.js

```diff
@@ -10,16 +10,20 @@ const GetApp = () => {
             <div className='back-drop'>
                 <div className='get-content'>
                     <div className='get-text'>
-                        <h2>Download the app</h2>
+                        <h2>Get the app</h2>
```

Comment:
'Download the app' is the agreed catch phrase (Check Figma staging)

Reply:
Forgot to close this PR.